skimage/filters/_homomorphic.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+from skimage import data, filters
+from skimage._shared.utils import _supported_float_type
+
+
+def homomorphic(filter_func, image, func_kwargs={}, eps=0):
+    """ Apply homomorphic filtering using a user-provided filter.
+
+    Apply a specified filtering function, `filter_func` to the logarithm of an
+    image. Specifically::
+
+    ``out = np.exp(filter_func(np.log(image)))``
```

Comment:
This docstring feels incomplete. It mentions a specific option, but does not mention that it should be customized. Is that the default given? What kinds of functions work? How should a user come up with these functions? How does the choice of function affect the outcome, etc.